content/ko/docs/concepts/configuration/secret.md

```diff
@@ -386,53 +235,23 @@ kubelet 동기화 기간 + 캐시 전파 지연만큼 길어질 수 있다.
 파드에서 {{< glossary_tooltip text="환경 변수" term_id="container-env-variables" >}} 형태로
 시크릿을 사용하려면 다음과 같이 한다.
 
-1. 시크릿을 생성(하거나 기존 시크릿을 사용)한다. 여러 파드가 동일한 시크릿을 참조할 수 있다.
 1. 사용하려는 각 시크릿 키에 대한 환경 변수를 추가하려면 
    시크릿 키 값을 사용하려는 각 컨테이너에서 파드 정의를 수정한다. 
    시크릿 키를 사용하는 환경 변수는 시크릿의 이름과 키를 `env[].valueFrom.secretKeyRef` 에 채워야 한다.
 1. 프로그램이 지정된 환경 변수에서 값을 찾도록 
    이미지 및/또는 커맨드 라인을 수정한다.
```

Comment:
원문 변경 사항이 좀 있네요 ㅎㅎ

```
1. For each container in your Pod specification, add an environment variable
   for each Secret key that you want to use to the
   `env[].valueFrom.secretKeyRef` field.
1. Modify your image and/or command line so that the program looks for values
   in the specified environment variables.
```

```suggestion
1. 파드 스펙 내의 각 컨테이너에 대해, 
   사용하려는 각 시크릿 키에 대한 환경 변수를 
   `env[].valueFrom.secretKeyRef` 필드에 추가한다.
1. 위에서 명시한 환경 변수의 값을 프로그램이 읽을 수 있도록 
   이미지 및/또는 커맨드 라인을 수정한다.
```

Reply:
앗! 제가 이 부분을 놓쳤었네요.
꼼꼼한 리뷰 정말 감사드립니다!!